win32/tests/malloc_multi_flex_int/malloc_multi_flex_int.c

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdint.h>
+
+#include "testrunnerswitcher.h"
+
+#include "c_pal/gballoc_hl.h"
+#include "c_pal/gballoc_hl_redirect.h"
+
+#include "c_pal/malloc_multi_flex.h"
+
```

Comment:
Why are these not unit tests?
How do we simulate malloc failing? We probably do not. If we did we would have caught the leak I spotted in the review ...